sql/static/country_codes.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+# Creates static tables for mapping between country codes and country names
+# by loading data from CSV files.
+
+cd "$(dirname "$0")"
+
+# List of ISO 3166 alpha-2 country codes and names generated via:
+# curl -sL https://datahub.io/core/country-list/r/data.csv | sed 's/\(.*\),\([A-Za-z]*\).*/\2,\1/' > country_codes.csv
+# See https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2
+
+
+## Load country_codes_v1
+
+COUNTRY_CODES_SCHEMA='
+[
+  {
+    "name": "code",
+    "description": "ISO 3166 alpha-2 country code",
+    "type": "STRING",
+    "mode": "REQUIRED"
+  },
+  {
+    "name": "name",
+    "description": "Official country name per ISO 3166",
+    "type": "STRING",
+    "mode": "REQUIRED"
+  }
+]
+'
+
+bq rm -f moz-fx-data-derived-datasets:static.country_codes_v1
```

Comment:
this will create a downtime period when the table is updated. we should instead use `bq load --replace` to ensure we don't have double-entries, and `bq update --description` to enforce the description